lib/modules/manager/pipenv/artifacts.spec.ts

```diff
@@ -24,7 +24,6 @@ import {
   extractEnvironmentVariableName,
   getMatchingHostRule,
 } from './artifacts';
-import type { PipfileLockSchema } from './schema';
```

Comment:
revert, we should still use it see other comments

Reply:
It doesn't exist anymore

Reply:
It was heavily refactored in the `detect-tools` package